staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -439,6 +439,15 @@ const (
 	//
 	// The annotation is added to a "Bookmark" event.
 	InitialEventsAnnotationKey = "k8s.io/initial-events-end"
+
+	// InitialEventsListBlueprintAnnotationKey is the name of the key
+	// where an empty, base64-encoded, versioned list is stored as a string.
```

Comment:
explain that the encoding matches the request encoding and can therefore be proto, json, cbor, (can it be the table format?)

Reply:
done, ptal.